### ISSUE TITLE
[FEATURE] Save Game/

### ISSUE THREAD

**Issue by MegaShinySnivy** [Phalcode/gamevault-backend#261]
**Is your feature request related to a problem? Please describe.**
Given that cloud syncing is GameVault+ only, I would like an option that does not require a subscription. 

**Describe the solution you'd like**
A way of syncing game saves or save states on emulators with a remote S3 bucket, like Minio, R2, B2, and so on.

**Describe alternatives you've considered**
Manual syncing via rsync or similar, but this would require separate system setup and distribution of keys. 

**Additional context**
N/A

**Comment by Alfagun74** [comment]
The Save-Game feature isn't available in GameVault yet. It was just an idea we discussed while talking about GameVault+. When we mentioned "Cloud"-Saves, we didn't intend to create a central server for everyone to upload games online (though we might consider it later). Instead, we're thinking about integrating this save-game feature and adding a paywall for the option to upload save-games to your self-hosted game vault backend server.

I opened up a ticket for the idea of savegames in general: https://github.com/Phalcode/gamevault-app/issues/374
